scripts/extract_policies.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+CHART_DIR=$1
+CHARTS_DIRS=`find $CHART_DIR -type d -exec test -e '{}'/values.yaml \; -print`
+POLICYLIST_FILENAME=policylist.txt
+TMP_IMAGE_FILE=/tmp/$POLICYLIST_FILENAME
+
+if [ -e $POLICYLIST_FILENAME ]; then
+	rm $POLICYLIST_FILENAME
+fi
+for chart in $CHARTS_DIRS; do
+	helm template --values $chart/values.yaml --set recommendedPolicies.enabled=true $chart/ | yq -r "..|.module?" | sort -u | grep -v "null"  > $TMP_IMAGE_FILE
```

Comment:
jq / yq can return nothing instead of null, no need to grep it afterwards - but result is the same so whatever you prefer

```suggestion
	helm template --values $chart/values.yaml --set recommendedPolicies.enabled=true $chart/ | yq -r "..|.module? // empty" | sort -u  > $TMP_IMAGE_FILE
```

Reply:
Thanks! I did not know that.